### ISSUE TITLE
Wrong weather data for Nowy Sacz (Poland)

### ISSUE THREAD

**Issue by Foxar** [chubin/wttr.in#260]
Weather data for Nowy Sacz, Poland seems incorrect. Wttr.in reports, for 11th of January:
`

Weather report: ~Nowy Sącz

     \   /     Sunny
      .-.      -19--11 °C     
   ― (   ) ―   ↘ 22 km/h      
      `-’      16 km          
     /   \     0.0 mm         
                                                       ┌─────────────┐                                                       
┌──────────────────────────────┬───────────────────────┤  Fri 11 Jan ├───────────────────────┬──────────────────────────────┐
│            Morning           │             Noon      └──────┬──────┘     Evening           │             Night            │
├──────────────────────────────┼──────────────────────────────┼──────────────────────────────┼──────────────────────────────┤
│    \  /       Partly cloudy  │    \  /       Partly cloudy  │     \   /     Clear          │     \   /     Clear          │
│  _ /"".-.     -21--14 °C     │  _ /"".-.     -20--13 °C     │      .-.      -21--14 °C     │      .-.      -23--16 °C     │
│    \_(   ).   → 16-18 km/h   │    \_(   ).   → 16-18 km/h   │   ― (   ) ―   → 16-24 km/h   │   ― (   ) ―   → 14-21 km/h   │
│    /(___(__)  20 km          │    /(___(__)  20 km          │      `-’      20 km          │      `-’      20 km          │
│               0.0 mm | 0%    │               0.0 mm | 0%    │     /   \     0.0 mm | 0%    │     /   \     0.0 mm | 0%    │
└──────────────────────────────┴──────────────────────────────┴──────────────────────────────┴──────────────────────────────┘
                                                       ┌─────────────┐                                                       
┌──────────────────────────────┬───────────────────────┤  Sat 12 Jan ├───────────────────────┬──────────────────────────────┐
│            Morning           │             Noon      └──────┬──────┘     Evening           │             Night            │
├──────────────────────────────┼──────────────────────────────┼──────────────────────────────┼──────────────────────────────┤
│     \   /     Sunny          │    \  /       Partly cloudy  │    \  /       Partly cloudy  │    \  /       Partly cloudy  │
│      .-.      -22--17 °C     │  _ /"".-.     -15--11 °C     │  _ /"".-.     -18--14 °C     │  _ /"".-.     -18--15 °C     │
│   ― (   ) ―   → 10-16 km/h   │    \_(   ).   → 9-17 km/h    │    \_(   ).   ↘ 11-15 km/h   │    \_(   ).   ↘ 7-10 km/h    │
│      `-’      20 km          │    /(___(__)  20 km          │    /(___(__)  20 km          │    /(___(__)  20 km          │
│     /   \     0.0 mm | 0%    │               0.0 mm | 0%    │               0.0 mm | 0%    │               0.0 mm | 0%    │
└──────────────────────────────┴──────────────────────────────┴──────────────────────────────┴──────────────────────────────┘
                                                       ┌─────────────┐                                                       
┌──────────────────────────────┬───────────────────────┤  Sun 13 Jan ├───────────────────────┬──────────────────────────────┐
│            Morning           │             Noon      └──────┬──────┘     Evening           │             Night            │
├──────────────────────────────┼──────────────────────────────┼──────────────────────────────┼──────────────────────────────┤
│    \  /       Partly cloudy  │    \  /       Partly cloudy  │     \   /     Clear          │     \   /     Clear          │
│  _ /"".-.     -19--16 °C     │  _ /"".-.     -13--10 °C     │      .-.      -16--13 °C     │      .-.      -19--15 °C     │
│    \_(   ).   ↘ 6-9 km/h     │    \_(   ).   ↘ 8 km/h       │   ― (   ) ―   ↙ 5 km/h       │   ― (   ) ―   ↖ 6 km/h       │
│    /(___(__)  20 km          │    /(___(__)  20 km          │      `-’      20 km          │      `-’      20 km          │
│               0.0 mm | 0%    │               0.0 mm | 0%    │     /   \     0.0 mm | 0%    │     /   \     0.0 mm | 0%    │
└──────────────────────────────┴──────────────────────────────┴──────────────────────────────┴──────────────────────────────┘
`

While it is barely below zero, about -4 degrees, as shown by many sources:

https://pogoda.interia.pl/prognoza-dlugoterminowa-nowy-sacz,cId,23590

**Comment by chubin** [comment]
Yes, it is possible, that our data source delivers wrong data for some locations.
We need to add some other data sources to fix this problem.

**Comment by chubin** [comment]
@Foxar Do you have this problem only if you don't specify any location?

**Comment by Foxar** [comment]
Prediction is now better, temperature measurement is at "-5 - -1 C" which is much closer to reality. There is no heavy snow however, as wttr.in reports, in fact there is no snowing, but it might be just the differences between areas of the location (Nowy Sacz).

So It'd appear that yes, It is fixed. I can comment further if I see any discrepancies.
